algo/src/main/java/com/hartwig/actin/algo/evaluation/tumor/TumorMetastasisEvaluator.java

```diff
@@ -7,9 +7,10 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-public class TumorUtil {
+public final class TumorMetastasisEvaluator {
```

Comment:
Consider adding a private constructor? I do consistently to avoid instantiation but also open for another approach if you prefer.

Reply:
It's just unnecessary, and I've done this consistently so far across ACTIN code. Other than that no strong reason.

If you and Paul feel it adds too little I am fine to drop it. Let me know!

Reply:
Fine!